### PR TITLE
update tests

### PR DESCRIPTION
**Pull request by nikpivkin**
Tests now uses a fixed version of db, java-db and a bundle of checks for data consistency.

### CODE REVIEW

**Comment by simar7** [comment]
@nikpivkin looks like tests are red. Did you forget to push some changes?

**Comment by nikpivkin** [comment]
@simar7 Tests are red, because there are not yet copies of the images of the dbs and the checks bundle created

File: .github/workflows/sync-trivy-db.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - uses: actions/checkout@v4
+
+      - name: Login to GitHub Packages Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Copy Trivy Checks
```
Comment (simar7) [inline .github/workflows/sync-trivy-db.yaml:22]:
```suggestion
      - name: Copy Trivy DB
```

File: .github/workflows/sync-trivy-java-db.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      - uses: actions/checkout@v4
+
+      - name: Login to GitHub Packages Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Copy Trivy Checks
```
Comment (simar7) [inline .github/workflows/sync-trivy-java-db.yaml:22]:
```suggestion
      - name: Copy Trivy Java DB
```

File: test/test.bats [hunk 3]
```diff
@@ -1,99 +1,123 @@
...
+      jq 'del(.runs[].tool.driver.version) | del(.runs[].originalUriBaseIds)' "$file" > tmp && mv tmp "$file"
+  fi
+}
+
+function remove_github_fields() {
+  local file="$1"
+  if [[ "$file" == *.gsbom  ]]; then
+      jq 'del(.detector.version) | del(.scanned) | del(.job) | del(.ref) | del(.sha)' "$file" > tmp && mv tmp "$file"
+  fi
+}
```
Comment (simar7) [inline test/test.bats:35]: nice this is a clever idea.

**Pull request merged**